### ISSUE TITLE
Some images missing?

### ISSUE THREAD

**Issue by DarrenCook** [superpowers-extra/fat-kevin-game#1]
I've loaded the zip file contents into my projects folder, restarted the server, and opened it up. When I run the game everything is very dark, and the floor/wall images are missing. Sprites, furniture, decorations, etc. are there. E.g. In the initial room the floor and secretaries counter are missing, but the computer and papers are on the invisible counter. I see the windows behind her, but not the walls.

The game plays fine. I can run around, fight and explore. The wall collisions still work, as do the doors - it is just that they are invisible.

This is on Linux (Mint 17), with superpowers 0.19.0-linux-x64.

When I run in debug mode, the network tab says 128/659 requests, 10.2MB/38MB transferred. I guess that is the problem?

Just tried the web version, in Chrome, and it said 38MB downloaded.

**Comment by elisee** [comment]
Hi and thanks for the report!
- Does the Web version look fine?
- Can you try running the source project in Chrome too (you can access your server at http://127.0.0.1:4237/ while it is running) and let me know if the game looks fine through Chrome?

Just trying to determine whether it's an Electron issue or more general one.

**Comment by DarrenCook** [comment]
It is fine (at that 127.0.0.1 URL) with either Firefox or Chrome. (BTW, I noticed Firefox does not have the sound effects - I'll take a closer look at that later.)

**Comment by DarrenCook** [comment]
P.S. I also tried exporting project, then loaded index.html into Firefox (works fine), and Chromium (stops immediately with  "COULD NOT LOAD PLUGINS LIST"). I could get chromium working from the commandline, though, with:
 `chromium-browser --disable-web-security index.html`
Anyway, in both cases, all images are fine.

**Comment by elisee** [comment]
Yeah when running with Chrome, you need to access the game through a web server for XHR to work (or use the flag trick you did).

OK so it looks like it could be an Electron-specific bug on Linux. Maybe you can figure out what asset refuses to load?

**Comment by DarrenCook** [comment]
The splash screen, and game scenario screen, come up fine, no errors in console. But then as we start the game proper there is lots of console activity, starting with:

```
THREE.WebGLShader: Shader couldn't compile.   (SupEngine.js:26906)
THREE.WebGLShader: gl.getShaderInfoLog() fragment 0:83(142): error: sampler arrays indexed with non-constant expressions is forbidden in GLSL 1.30 and later  (SupEngine.js:26912)
   ....lots of similar complaints...
SupEngine.js:23150 WebGL: INVALID_OPERATION: useProgram: program not valid
   ....lots more...
```

(BTW, the similar-looking "discover-superpowers-game" is not having this issue.)

**Comment by elisee** [comment]
Looks like we might be doing something shady (get it? Because shaders! Hilarious!) and some drivers are lenient or others are not.

An approach to fixing this would be to setup a breakpoint at `SupEngine.js:26906` and look up the callstack to find out what shader asset is causing problems.
